src/AppliesHttpQuery.php

```diff
@@ -25,7 +25,7 @@ public function scopeHttpQuery(Builder $query): Builder
         }
 
         if ($request->has('search')) {
-            $this->applySearch($query, $request->get('search'));
+            $this->applySearch($query, $request->get('search'), $request->get('searchFields'));
```

Comment:
Will fail due typehinting of `applySearch` when `searchFields` is not present in the request and returns `null`. Replace with `$request->get('searchFields', '')`.